understanding/20/images-of-text.html

```diff
@@ -130,139 +120,54 @@ <h2>Examples of Images of Text</h2>
          <dt>Customizable font settings in images of text</dt>
          <dd>A Web site allows users to specify font settings and all images of text on the site
                are then provided based on those settings.</dd>
+         <dt>The text in an image is reproduced in text</dt>
```

Comment:
Missing period.  Sorry to say, "reproduced in text" reads awkwardly.

> The text in an image is also provided as text.

Reply:
though this goes slightly against the point made here https://github.com/w3c/wcag/pull/4021#issuecomment-2379684390

Reply:
```suggestion
         <dt>The text in an image is also provided as text.</dt>
```